### DISCUSSION TITLE
Progress Report 25–29 July

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#66] [Progress Report]
🚴🏻 **Progress**

- Preliminary map of vector data (#65)

😖 **Regressions**

None

💬 **To Discuss**

- Access to more data files (preferably through AWS)
- #44 
- #67 

📆 **Next Week**

- Visualize scalar components of vector data

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**

- The application is available on an internal AWS URL

😖 **Regressions**

- Cluster ingress broke for the dev cluster when we switched the frontend back to Nginx

💬 **To Discuss**

- Merging dev k8s manifests into a sub-directory of this proejct
- Accessing data files via S3 instead of the filesystem

📆 **Next Week**

- Fix dev cluster ingress
- Help VLab with an official URL
- Contribute to VLab discussions around using Helm or Kustomize to package the app for deployment
